apps/console/src/init/app-utils.ts

```diff
@@ -223,7 +223,9 @@ export const AppUtils: any = (function() {
             return {
                 __experimental__platformIdP: _config.__experimental__platformIdP,
                 accountApp: {
+                    clientID: _config.accountApp.clientID,
                     commonPostLogoutUrl : commonPostLogoutUrl,
+                    impersonationRoleName: _config.accountApp.impersonationRoleName,
```

Comment:
_config is to get dynamic values. since impersonationrolename is something that wouldn't change, we shouldn't use it here, IMO.